config-model-api/src/test/java/com/yahoo/config/application/api/DeploymentSpecTest.java

```diff
@@ -1782,18 +1787,24 @@ public void cloudAccount() {
                 </deployment>
                 """;
         DeploymentSpec spec = DeploymentSpec.fromXml(r);
-        assertEquals(Optional.of(CloudAccount.from("100000000000")), spec.cloudAccount());
-        assertCloudAccount("800000000000", spec.requireInstance("alpha"), prod, "us-east-1");
-        assertCloudAccount("200000000000", spec.requireInstance("beta"), prod, "us-west-1");
-        assertCloudAccount("600000000000", spec.requireInstance("beta"), staging, "");
-        assertCloudAccount("700000000000", spec.requireInstance("beta"), perf, "");
-        assertCloudAccount("200000000000", spec.requireInstance("beta"), dev, "");
-        assertCloudAccount("300000000000", spec.requireInstance("main"), prod, "us-east-1");
-        assertCloudAccount("100000000000", spec.requireInstance("main"), prod, "eu-west-1");
-        assertCloudAccount("400000000000", spec.requireInstance("main"), dev, "");
-        assertCloudAccount("500000000000", spec.requireInstance("main"), test, "");
-        assertCloudAccount("100000000000", spec.requireInstance("main"), staging, "");
-        assertCloudAccount("default", spec.requireInstance("beta"), prod, "us-west-2");
+        assertEquals(Map.of(AWS, CloudAccount.from("100000000000"),
+                            GCP, CloudAccount.from("gcp:foobar")), spec.cloudAccounts());
+        assertCloudAccount("800000000000", spec, AWS, "alpha", prod,    "us-east-1");
+        assertCloudAccount("",             spec, GCP, "alpha", prod,    "us-east-1");
```

Comment:
Empty string is so ugly though.. Why not 'default'?

Reply:
Both work ... but empty string here means custom accounts for _no_ clouds, because the semantics of the string is that each cloud with a custom account is listed, comma separated, in the format accepted by `CloudAccount.from`, which is documentable. If we want, we can document that `default` is an alias for that.